src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1113,7 +1113,9 @@ class DocBaseViewer extends BaseViewer {
      * @private
      * @return {void}
      */
-    regionClickHandler() {}
+    regionClickHandler({ activeControl }) {
```

Comment:
Is this needed? Wouldn't this always call with 'region'?

Reply:
> Is this needed? Wouldn't this always call with 'region'?

It might be `null`. I thought this would simplify togglemode method in box-annotations